syntax = "proto3";

message PlayerLogin {
	string account = 1;
	string password = 2;
}
message PlayerLoginAck {
	uint32 session_key = 1;	
	enum CheckResult
    {
        Pass = 0;
        Refuse = 1;;
    }
	CheckResult result = 2;
}
message LoginLogic {
	uint32 session_key = 1;
}
message PlayerRegsiter {
	string account = 1;
	string password = 2;
}

message BenchTest{
	uint32 num = 1;
}